apps/condo/domains/miniapp/schema/B2BAppContext.js

```diff
@@ -6,14 +6,59 @@ const { Relationship } = require('@keystonejs/fields')
 const get = require('lodash/get')
 
 const { Json } = require('@open-condo/keystone/fields')
+const { getLogger } = require('@open-condo/keystone/logging')
 const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
-const { GQLListSchema } = require('@open-condo/keystone/schema')
+const { GQLListSchema, find } = require('@open-condo/keystone/schema')
 const { webHooked } = require('@open-condo/webhooks/plugins')
 
 const access = require('@condo/domains/miniapp/access/B2BAppContext')
 const { NO_CONTEXT_STATUS_ERROR, CONTEXT_FINISHED_STATUS } = require('@condo/domains/miniapp/constants')
 const { STATUS_FIELD, getStatusResolver, getStatusDescription } = require('@condo/domains/miniapp/schema/fields/context')
-const { createDefaultB2BAppRoles, deleteB2BAppRoles } = require('@condo/domains/miniapp/tasks')
+const { deleteB2BAppRoles } = require('@condo/domains/miniapp/tasks')
+const { B2BAppRole } = require('@condo/domains/miniapp/utils/serverSchema')
+
+const logger = getLogger('miniapp/createDefaultB2BAppRoles')
+
+async function createDefaultB2BAppRoles (appId, organizationId, sender, context) {
```

Comment:
Why did you remove this logic from task?

Reply:
I've realised, that we'll need to do it in sync. Since on a client we click a "Connect" button, which creates a context. And after receiving the response, we should be able to open an app (which is not possible before roles are created)

So we can do a polling to get notified when we can open an app (which is hard and can take a few minutes in case were queue is overwhelmed), or just create it "in sync manner" since response is not returned until `afterChange` is finished working, so we can guarantee on client, that everything is ready to use